modules/software-extensibility/pages/extensions-sdk.adoc

```diff
@@ -0,0 +1,28 @@
+= Extensions Source Development Kits(SDK)
+:description: Several Source Development Kits are provided so that you can extend your automation project
```

Comment:
Lol 


And by the ways archetypes are just archetypes, it is not really an SDK... But I know the product team want to use this word. 
```suggestion
:description: Several Software Development Kits are provided so that you can extend your automation project
```